scripts/zones/Halvung/mobs/Big_Bomb.lua

```diff
@@ -2,11 +2,17 @@
 -- Area: Halvung
 --   NM: Big Bomb
 -----------------------------------
+mixins = { require('scripts/mixins/families/bomb_grow') }
+-----------------------------------
 ---@type TMobEntity
 local entity = {}
 
-entity.onMobInitialize = function(mob)
-    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 900)
+entity.onMobSpawn = function(mob)
+    mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 180)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:addImmunity(xi.immunity.DARK_SLEEP)
+    mob:addImmunity(xi.immunity.STUN)
+    mob:setMod(xi.mod.SILENCERES, 80) -- Approximation - requires further experimentation
```

Comment:
This modifier is used for "Resist Silence" job trait, almost exclusively.
Did the message return `Resist! The big bomb resisted...` When casting Silence on the mob? (The `Resist!` at the beggining is whats important.)

Reply:
Upon re-reviewing the captures done by Siknawz, it appears Big Bomb is also immune to terror, and disease.

However, with silence it simply returned Resists... not Resist!

I will commit terror and disease immunities. Am I using the appropriate mod for silence? Or should I be using SILENCE_MEVA

Reply:
I edited my original message before I saw your answer, but I'll repeat it.

Use SILENCE_MEVA at a 100-200 value until we get status effect resistance ranks implemented

Reply:
All requested changes made <3